e2e/test_operator_upgrades/operator_upgrades_test.go

```diff
@@ -376,6 +379,22 @@ var _ = Describe("Operator Upgrades", Label("e2e"), func() {
 			)
 			Expect(err).NotTo(HaveOccurred())
 
+			// Wait for the server process to restart.
+			time.Sleep(40 * time.Second)
```

Comment:
Are those changes intended to be in this PR? They seem to be unrelated to the initial changes "test upgrading a cluster when no storage processes are restarted"?

Reply:
Adjusted the scope of this PR to cover testing multiple upgrade scenarios.